witnet/prices/e/EthUsdPrice.js

```diff
@@ -1,12 +1,5 @@
 import * as Witnet from "witnet-requests"
 
-// Retrieve ETH/USD-6 price from Binance.US
-const binance = new Witnet.Source("https://api.binance.US/api/v3/trades?symbol=ETHUSD")
-  .parseJSONMap()
-  .getFloat("price")
-  .multiply(10 ** 6)
-  .round()
-
```

Comment:
Did you remove this one?

Reply:
Sorry, the change on witnet/prices/e/EthUsdPrice.js was not included in the PR. The corrected Binance.us source in eth/usd was included within the bytecode change in migrations/witnet-queries.json